projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.ts

```diff
@@ -12,9 +13,11 @@ import {PaletteColorizer} from '../colorizer.model';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ColorPaletteEditorComponent implements OnInit {
-    // Output and Input for communication with the parent "raster-symbology-editor" component
-    @Input() symbology!: RasterSymbology;
-    @Output() symbologyChanged: EventEmitter<RasterSymbology> = new EventEmitter();
+    @Input() symbology!: RasterSymbology; // Symbology to use for creating color tabs
+    @Output() symbologyChanged: EventEmitter<RasterSymbology> = new EventEmitter(); // Symbology altered through color tab inputs
```

Comment:
it is uncommon to put doc-comments behind properties, but you put it rather on top of them

Reply:
Put the comments on top in [7fbcc6e](https://github.com/geo-engine/geoengine-ui/pull/614/commits/7fbcc6edc42516577798dfccc44d9fedc99f57fd)